anton-melyakov/homework1/CV anton/cv frontpage.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
```

Comment:
Avoid using file names with blank spaces. It can lead to problems across file systems and web browsers and usually it is uncomfortable to work with urls containing white spaces : )